drc_cmis/client.py

```diff
@@ -187,8 +187,7 @@ def update_document(
         content_filename = data.get("bestandsnaam") or cmis_doc.bestandsnaam
 
         try:
-            if content is not None:
-                cmis_doc.update_content(content, content_filename)
+            cmis_doc.update_content(content or "", content_filename)
```

Comment:
can you add a comment that we deliberately allow this for the "large file uploads mechanism" (i.e. this is intentional)?